workflows/toast_sim_ground.py

```diff
@@ -309,6 +314,9 @@ def main():
             out_model="el_noise_model",
         ),
         toast.ops.PointingDetectorSimple(name="det_pointing_azel", quats="quats_azel"),
+        # In the future, `det_weights_azel` may be a dedicated operator that does not
+        # expand pixel numbers but just Stokes weights
+        toast.ops.PointingHealpix(name="det_weights_azel", mode="IQU"),
```

Comment:
Note that unless the user specifies `--det_weights_azel.weights some_other_string`, the default name of the detector weights computed by this operator (`weights`) is the same as that used by the `pointing` operator below.  Perhaps we could set `weights="weights_azel"` or similar in the constructor?

Reply:
Done.